src/Features/CSharp/Portable/CodeFixes/Nullable/CSharpDeclareAsNullableCodeFixProvider.cs

```diff
@@ -140,6 +148,24 @@ private static TypeSyntax TryGetDeclarationTypeToFix(SyntaxNode node)
             }
 
             return null;
+
+            TypeSyntax tryGetSingleTypeArgument(TypeSyntax type)
```

Comment:
"Try"

Reply:
That intentional. Same naming convention as locals.

Reply:
IDE convention is that local functions are named like methods :)